### PR TITLE
Added Deploy to heroku button

### PR DESCRIPTION
**Pull request by WizzyGeek**
- Simplified the deploy process
- Added app.json
_ignore #1 mistakenly pulled to master_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,16 @@ This branch will automatically download the Lavalink.jar, but it may not be late
...
 
-**Using Github Deploy:**
+## Using this button (The easy way)
+##### Note
+You can enable automatic deploys to update the app when commits are added to the auto branch (see step 5 to 9).
+Also automatic deploys will restart your server so enable it on your own risk, no author is responsible for any damages that may occur due to auto matic deploys! or simply you may check the auto branch for commits and deploy them manually
+### Info 
+Buildpacks should be added automatically, you may modify the `PASS` variable during setup to change the password.
+### Deploy
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/KareemRS/heroku-lavalink/tree/auto)
```
Comment (WizzyGeek) [inline README.md:13]: This is the main addition cant do for master since lavalink.jar is not downloaded

File: README.md [hunk 2]
```diff
@@ -5,7 +5,16 @@ This branch will automatically download the Lavalink.jar, but it may not be late
...
+
+## Using Github Deploy: (The hard way but more control)
```
Comment (WizzyGeek) [inline README.md]: changed formatting to match the rest of README

**Comment by WizzyGeek** [comment]
Tests failed

**Comment by karyeet** [comment]
I was able to successfully deploy with the button, but tell me when you are ready.

**Comment by WizzyGeek** [comment]
You can merge it now. [also i need little help, how do you prevent idle connection on heroku?]

**Comment by karyeet** [comment]
You'll have to use uptimerobot or statuscake and have them ping your project url. Both are free to sign up and use. Set to 5m pings.

**Pull request merged**